config/crd/bases/management.project-flotta.io_edgeworkloads.yaml

```diff
@@ -7207,6 +7207,16 @@ spec:
                 required:
                 - spec
                 type: object
+              securityContext:
+                description: EdgeWorkloadSecurityContext holds workload-level security
+                  attributes
+                properties:
+                  runByRoot:
```

Comment:
At the [API](https://github.com/project-flotta/flotta-operator/pull/272/files#diff-8b1949772e223a1da6a2049ada2733fa506410975b241cf86cf44c7a8665bc62R782) level this field is called `RunAsRoot`. Why the difference? (I like `RunAsRoot` since it doesn't reveal information of how this will be executed)

Reply:
Personally, I would use the same convention as `podman` `rootful`. my 2 cents

Reply:
By doing that you're exposing implementation details and needs to be populated on each workload. Having a boolean with false by default simplifies the workload spec since the lack of it in the manifest will always mean that it is run by the flotta user, which is by probably the more generic use case.